### DISCUSSION TITLE
Error during UDM boot script install

### DISCUSSION THREAD

**Discussion by GigaBypass9876** [unifi-utilities/unifi-common#380] [Q&A] [answered]
UDM Pro SE with firmware 2.4.10
I installed UDM Boot Script with the remote install script but when I checked status, it was not running.  Here is output:

root@UDM-SE:~# systemctl status udm-boot
udm-boot.service - Run On Startup UDM
Loaded: loaded (/etc/systemd/system/udm-boot.service; enabled; vendor preset: enabled)
Active: failed (Result: exit-code) since Fri 2022-06-17 08:11:44 EDT; 1min 3s ago
Process: 20428 ExecStart=/bin/bash -c mkdir -p /mnt/data/on_boot.d && find -L /mnt/data/on_
CPU: 1.934s
Jun 17 08:11:44 UDM-SE bash[20428]: /mnt/data/on_boot.d/pi_hole_update.sh: line 29: --hostnam
Jun 17 08:11:44 UDM-SE bash[20428]: /mnt/data/on_boot.d/pi_hole_update.sh: line 30: -e: comma
Jun 17 08:11:44 UDM-SE bash[20428]: /mnt/data/on_boot.d/pi_hole_update.sh: line 31: -e: comma
Jun 17 08:11:44 UDM-SE bash[20428]: /mnt/data/on_boot.d/pi_hole_update.sh: line 32: -e: comma
Jun 17 08:11:44 UDM-SE bash[20428]: /mnt/data/on_boot.d/pi_hole_update.sh: line 33: -e: comma
Jun 17 08:11:44 UDM-SE bash[20428]: [96B blob data]
Jun 17 08:11:44 UDM-SE systemd[1]: udm-boot.service: Control process exited, code=exited, sta
Jun 17 08:11:44 UDM-SE systemd[1]: udm-boot.service: Failed with result 'exit-code'.
Jun 17 08:11:44 UDM-SE systemd[1]: Failed to start Run On Startup UDM.
Jun 17 08:11:44 UDM-SE systemd[1]: udm-boot.service: Consumed 1.934s CPU time.
lines 1-16/16 (END)

root@UDM-SE:~# journalctl -xe
Jun 17 08:18:48 UDM-SE systemd[1]: udm-boot.service: Failed with result 'exit-code'.
-- Subject: Unit failed
-- Defined-By: systemd
-- Support: https://www.debian.org/support
-- The unit udm-boot.service has entered the 'failed' state with result 'exit-code'.
Jun 17 08:18:48 UDM-SE systemd[1]: Failed to start Run On Startup UDM.
-- Subject: A start job for unit udm-boot.service has failed
-- Defined-By: systemd
-- Support: https://www.debian.org/support
-- A start job for unit udm-boot.service has finished with a failure.
-- The job identifier is 919 and the job result is failed.

NOTE, I tried the installation again this morning, and got this message in the output:
Job for udm-boot.service failed because the control process exited with error code.

Am I doing something wrong?

**Comment by GigaBypass9876** [comment]
Has anyone with a UDM-Pro SE gotten this error? Any thoughts on how to remedy? Thanks!

root@UDM-SE:~# journalctl -xe
Jun 17 08:18:48 UDM-SE systemd[1]: udm-boot.service: Failed with result 'exit-code'.
-- Subject: Unit failed
-- Defined-By: systemd
-- Support: https://www.debian.org/support
-- The unit udm-boot.service has entered the 'failed' state with result 'exit-code'.

**Reply by boostchicken** [reply]
Still having issues?

**Comment by GigaBypass9876** [comment]
Thanks for the email.You can close this #380. I do have a remaining issue
under the discussion "Pihole/UDM-boot.service error (exit code 123?) on UDM
Pro-SE #388", where I get ERRO[0011] error loading cached network config:
network "dns" not found in CNI cache. Any troubleshooting suggestions there
would be highly appreciated.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by boostchicken** [comment, marked as answer]
ok
